backend/server/controllers/experiments.js

```diff
@@ -85,6 +86,9 @@ module.exports = {
 
         return Experiment
             .findById(req.params.id, {
+                include: [
+                    {all: true}
```

Comment:
Does this include the experiment's questionnaires?

Reply:
Yes, this is the way that I got the many-to-many includes to work.

See: https://github.com/ianmunrobot/1702-express-review/blob/master/routes/puppies.js#L34

Reply:
Awesome. Thanks!